src/ucp/core/ucp_request.h

```diff
@@ -209,8 +209,10 @@ struct ucp_request {
                 } disconnect;
 
                 struct {
-                    ucp_worker_h          ucp_worker;     /* UCP worker where discard UCT EP operation
-                                                           * submitted on */
+                    uct_worker_cb_id_t    cb_id;          /* UCT worker callback ID to schedule
```

Comment:
IMO better to use one-shot callback, this way no need to store id and light weight on "likely" path

Reply:
done